src/fee-accounts/containers/account-transactions/account-transactions.component.html

```diff
@@ -7,5 +7,10 @@ <h1 class="govuk-body govuk-!-font-size-48 govuk-!-font-weight-bold">
   </div>
   <div class="hmcts-page-heading__actions-wrapper"></div>
 </div>
-<!--<pre>{{accountTransactions$ }}</pre>-->
-<app-govuk-table [columnConfig]="columnConfig" [rows]="accountTransactions$ | async" [firstCellIsHeader]="false" [caption]="''"></app-govuk-table>
+<app-govuk-table
+  *ngIf="!(loading$ | async) else loading"
```

Comment:
this should be hooked on the reducer loading property

Reply:
? This is hooked on the reducer. it is a selector that get Boolean if the data is loaded or not.

Reply:
sorry my bad you are right